documentation/book/assembly-kafka-bridge-authentication.adoc

```diff
@@ -0,0 +1,20 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-bridge.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-bridge-authentication-{context}']
+
+= Connecting to Kafka brokers with Authentication
+
+By default, Kafka Bridge will try to connect to Kafka brokers without authentication.
+Authentication is enabled through the `KafkaBridge` resources.
+
+include::con-kafka-bridge-authentication.adoc[leveloffset=+1]
+
+include::proc-configuring-kafka-bridge-authentication-tls.adoc[leveloffset=+1]
+
+include::proc-configuring-kafka-bridge-authentication-scram-sha-512.adoc[leveloffset=+1]
```

Comment:
Should we have a procedure for SASL PLAIN as well?

Reply:
'To configure Kafka Bridge to use SASL-based PLAIN authentication' is inside con-kafka-bridge-authentication.adoc. We should split it out at some stage.